packages/web/components/Player/Filter/PlayerFilter.tsx

```diff
@@ -67,6 +68,18 @@ export const PlayerFilter: React.FC<Props> = ({
     sortOptionsMap[SortOption.SEASON_XP],
   );
 
+  // Get ADDRESS from ENS, temporary.
+
+  useEffect(() => {
+    const resolveName = async () => {
+      if (search.includes('.')) {
+        const address = await getAddressFromName(search);
+        setQueryVariable('search', `%${address!}%`);
```

Comment:
Since the address should match exactly, I don't think you need the wildcards.

```suggestion
        if (address) {
          setQueryVariable('search', address);
        }
```

Reply:
In fact, now that I think about it, including the wildcards on an address would allow an attacker to potentially create a name that is a superstring of an ETH address and have it inappropriately match.

Reply:
I resolved this by following your suggestion, removed the *%